astropy/cosmology/core.py

```diff
@@ -99,13 +99,13 @@ class FLRW(Cosmology):
         Neff. Usually this means you must provide three neutrino masses unless
         you are considering something like a sterile neutrino.
 
-    name : str
-        Optional name for this cosmological object.
-
-    Ob0 : float
+    Ob0 : float or None
```

Comment:
Yeah, all right, fair enough.  The nomenclature is perhaps breaking down a bit here...